src/Microsoft.TestPlatform.Client/DesignMode/DesignModeClient.cs

```diff
@@ -149,8 +149,9 @@ private void ProcessRequests(ITestRequestManager testRequestManager)
 
                         case MessageType.ExtensionsInitialize:
                             {
+                                // Do not filter the Editor/IDE provided extensions by name
```

Comment:
This is required because VSIX and other inbox extensions don't conform to the file naming logic (compat with TPv1).